ros2_controllers_test_nodes/ros2_controllers_test_nodes/publisher_forward_position_controller.py

```diff
@@ -0,0 +1,81 @@
+# Copyright 2022 Stogl Robotics Consulting UG (haftungsbeschränkt)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Author: Denis Štogl
+#
+
+import rclpy
+from rclpy.node import Node
+
+from std_msgs.msg import Float64MultiArray
+
+
+class PublisherForwardPosition(Node):
+    def __init__(self):
+        super().__init__("publisher_forward_position_controller")
+        # Declare all parameters
+        self.declare_parameter("controller_name", "forward_position_controller")
+        self.declare_parameter("wait_sec_between_publish", 5)
+        self.declare_parameter("goal_names", ["pos1", "pos2"])
+
+        # Read parameters
+        controller_name = self.get_parameter("controller_name").value
+        wait_sec_between_publish = self.get_parameter("wait_sec_between_publish").value
+        goal_names = self.get_parameter("goal_names").value
+
+        # Read all positions from parameters
+        self.goals = []
+        for name in goal_names:
+            self.declare_parameter(name)
+            goal = self.get_parameter(name).value
+            if goal is None or len(goal) == 0:
+                raise Exception(f'Values for goal "{name}" not set!')
+
+            float_goal = []
+            for value in goal:
+                float_goal.append(float(value))
```

Comment:
```suggestion
            float_goal = [float(value) for value in goal]
```
should be much faster